examples/async.py

```diff
@@ -5,50 +5,48 @@
 # 10 minutes. This is useful when creating many documents in parallel, or very
 # large documents with lots of assets.
 #
-# DocRaptor supports many options for output customization, the full list is
-# https://docraptor.com/documentation/api#api_general
+# DocRaptor supports many CSS and API options for output customization. Visit
+# https://docraptor.com/documentation/ for full details.
 #
 # You can run this example with: python async.py
 
 import docraptor
 import time
-import shutil
 
 doc_api = docraptor.DocApi()
-doc_api.api_client.configuration.username = 'YOUR_API_KEY_HERE' # this key works for test documents
-# doc_api.api_client.configuration.debug = True
```

Comment:
I don't think this is blocking, but I'm curious if you know why this was in here previously? It seems like maybe it would provide more info in error cases, but I'm not actually sure off the top of my head.

Reply:
It was in a lot of our historic examples and I think people just keep using it out of habit. But it didn't really do anything useful, if I recall. I think it increases Prince's debugging logging, but not in a helpful-to-users way.